source/developers/file-transfer-protocol.rst

```diff
@@ -0,0 +1,226 @@
+=================================
+
+.. default-domain:: minio
+
+.. container:: extlinks-video
+
+   - `File Transport Using FTP and SFTP with MinIO <https://www.youtube.com/watch?v=lNZyL8wD-lI>`__
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. versionadded:: MinIO RELEASE.2023-04-20T17-56-55Z
+
+Overview
+--------
+
+You can use the File Transfer Protocol (``FTP``) to interact with the objects on a MinIO deployment.
+
+You must specifically enable FTP or SFTP when starting the server.
+Enabling either server type does not affect other MinIO features.
+
+This page uses the abbreviation ``FTP`` throughout, but you can use any of the supported FTP protocols described below.
+
+Supported Protocols
+~~~~~~~~~~~~~~~~~~~
+
+When enabled, MinIO supports FTP access over the following protocols:
+
+- SSH File Transfer Protocol (``SFTP``)
+
+  SFTP is defined by the Internet Engineering Task Force (IETF) as an extension of SSH 2.0.
+  SFTP allows file transfer over SSH for use with :ref:`Transport Layer Security (TLS) <minio-tls>` and virtual private network (VPN) applications.
+
+  Your FTP client must support SFTP.
+
+- File Transfer Protocol over SSL/TLS (``FTPS``)
+  
+  ``FTPS`` allows for encrypted FTP communication with TLS certificates over the standard FTP communication channel.
+  ``FTPS`` should not be confused with ``SFTP``, as ``FTPS`` does not communicate over a Secure Shell (``SSH``).
+
+  Your FTP client must support FTPS.
+
+- File Transfer Protocol (``FTP``)
+  
+  Unencrypted file transfer.
+
+  MinIO does **not** recommend using unencrypted FTP for file transfer.
+
+Supported Commands
+~~~~~~~~~~~~~~~~~~
+
+When enabled, MinIO supports the following ``ftp`` operations:
+
+- ``get``
+- ``put``
+- ``ls``
+- ``mkdir``
+- ``rmdir``
+- ``delete``
+
+MinIO does not support either ``append`` or ``rename`` operations.
+
+Considerations
+--------------
+
+Versioning
+~~~~~~~~~~
+
+You cannot use ``FTP`` to read specific :ref:`object versions <minio-bucket-versioning>` other than the latest version.
+
+- For read operations, MinIO only returns the latest version of the requested object(s) to the ftp client.
+- For write operations, MinIO applies normal versioning behavior for matching object names.
+
+Use an S3 API Client, such as the :ref:`MinIO Client <minio-client>`.
+
+Authentication and Access
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``FTP`` access requires the same authentication as any other S3 client.
+MinIO supports the following authentication providers:
+
+- :ref:`MinIO IDP <minio-internal-idp>` users and their service accounts
+- :ref:`Active Directory/LDAP <minio-external-identity-management-ad-ldap>` users and their service accounts
+- :ref:`OpenID/OIDC <minio-external-identity-management-openid>` service accounts
+
+:ref:`STS <minio-security-token-service>` credentials **cannot** access buckets or objects over FTP.
+To use STS credentials to authenticate, you must use an S3 API client or port.
```

Comment:
:thinking:  Oh wait.

If we can't use STS , then we can't use OIDC/AD/LDAP credentials outside of creating access keys.

So we should state that outright for the latter two.

Reply:
We can use service accounts. STS can't be used because STS requires 3 things instead of two things that ftp client supports.